starport/services/network/publish.go

```diff
@@ -107,6 +108,7 @@ func (n Network) Publish(ctx context.Context, c Chain, options ...PublishOption)
 		if err != nil {
 			return 0, 0, 0, err
 		}
+		chainID = fmt.Sprintf("%s-1", chainID)
```

Comment:
`chainID` here might be already in `x-1` format. I think we should check that first and then add `-1` if not exists.

Reply:
Also maybe it makes sense implement a `network.ChainID()` func and move this logic there?

Reply:
> `chainID` here might be already in `x-1` format. I think we should check that first and then add `-1` if not exists.

We can't fetch the last chain id from the network quickly. It's tricky to increment one. I added logic to create with a random number.